components/tools/OmeroPy/src/omero/plugins/admin.py

```diff
@@ -1789,7 +1791,7 @@ def ports(self, args, config):
             " regenerated. Use the omero.ports.xxx configuration properties"
             " instead.")
 
-    @admin_only()
+    @admin_only(AdminPrivilegeReadSession)
```

Comment:
I appreciate that this represents the current state of 5.4.x but I'm still worried that A) this is confusing to users and developers (or at least to me) and B) that this isn't future-proof unless we overload `ReadSession` to be equivalent with `FullAdmin`. I've added a card for us to further discuss the definition of `FullAdmin`: https://trello.com/c/4dvUYSjS/126-definition-of-ultimate-admin